libs/ledger-live-common/src/featureFlags/defaultFeatures.ts

```diff
@@ -64,4 +64,10 @@ export const defaultFeatures: DefaultFeatures = {
   counterValue: {
     enabled: false,
   },
+  llmUsbFirmwareUpdate: {
```

Comment:
this was missing because the typing was not strict enough.
see https://github.com/LedgerHQ/ledger-live/pull/743/files#r930990175